Configuration/Services.yaml

```diff
@@ -20,6 +20,18 @@ services:
         identifier: 'cart--cart--update-currency'
         event: Extcode\Cart\Event\Cart\UpdateCurrency
 
+  Extcode\Cart\EventListener\Mail\AttachmentFromOrderItem:
+    tags:
+      - name: event.listener
+        identifier: 'cart--mail--attachment-from-order/item'
+        event: Extcode\Cart\Event\Mail\AttachmentEvent
```

Comment:
According to https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/11.3/Feature-94345-AutoDetectEventTypes.html the `event` attribute is not needed any longer for v12. This release will be v12+ won't it? (valid for all EventListener entries)

Reply:
Without a given method it can't retrieve the correct event. I'll change this for new EventListeners.

Reply:
Unfortunately I do not understand your comment :-/ Which method do you mean?